dice_ml/data_interfaces/public_data_interface.py

```diff
@@ -118,6 +115,11 @@ def _validate_and_set_continuous_features(self, params):
             raise ValueError(
                 "should provide the name(s) of continuous features in the data as a list")
 
+    def set_continuous_feature_indexes(self, query_instance):
```

Comment:
This function should also be defined in private_data_interface.py as any of the explainer can be called on top of either public_data_interface or private_data_interface. It might be better that we have this function defined in base_data_interface.py from which public/private data interfaces are inherited.

Reply:
That's a good idea

Reply:
Thanks for moving this into base class.